bdd100k/common/utils.py

```diff
@@ -3,27 +3,42 @@
 import glob
 import os
 import os.path as osp
+from itertools import groupby
 from typing import List
 
+import numpy as np
+from PIL import Image
 from scalabel.label.io import load as load_bdd100k
 from scalabel.label.typing import Frame
 
+from ..common.typing import InstanceType
+
 DEFAULT_COCO_CONFIG = osp.join(
     osp.dirname(osp.abspath(__file__)), "configs.toml"
 )
 
 
-def list_files(inputs: str) -> List[List[str]]:
```

Comment:
Split the list_files -> List[List[str]]  func into funcs:
list_files -> List[str]
group_and_sort_files -> List[List[str][
By which, it is suitable for both ins_seg and seg_track bitmask files listing